cluster-autoscaler/cloudprovider/konvoy/konvoy.go

```diff
@@ -142,6 +132,9 @@ func (konvoy *KonvoyCloudProvider) GetResourceLimiter() (*cloudprovider.Resource
 // Refresh is called before every main loop and can be used to dynamically update cloud provider state.
 // In particular the list of node groups returned by NodeGroups can change as a result of CloudProvider.Refresh().
 func (konvoy *KonvoyCloudProvider) Refresh() error {
+	if err := konvoy.konvoyManager.forceRefresh(); err != nil {
+		return err
+	}
 	return nil
```

Comment:
If you don't wrap the error might as well simplify the code.

```suggestion
	return konvoy.konvoyManager.forceRefresh()
```